src/Config/PrefetchOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
+{
+    public class PrefetchOptions
+    {
+        public ushort PrefetchSize { get; set; }
+
+        public ushort PrefetchCount { get; set; }
```

Comment:
I think you should make these ints - the rest of our integral config are int

Reply:
Also, are there any limits that need to be applied to these? What are the limits RabbitMQ enforces, and what happens if someone configures over those limits?

Reply:
RabbitMQ client accepts only ushort though..

Reply:
And weirdly I haven't found official docs specifying any limits on them :( 
https://www.cloudamqp.com/blog/2017-12-29-part1-rabbitmq-best-practice.html
https://www.rabbitmq.com/consumer-prefetch.html
I can try to see if I can dig up some more, but I don't think we should enforce something that isn't officially documented.